tasks/release-git-rebase.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: release-git-rebase
+spec:
+  resources:
+    inputs:
+      - name: git-source-repository
+        type: git
+  params:
+    - name: upstream-url
```

Comment:
as upstream-url info are only avialable in pull request, repository payloads and few more.
we should create another step or task to gather the _upstream repository ssh url_ .

it could directly use the `get` method on repository , and gather `parent->ssh_url`